Cargo.toml

```diff
@@ -1,10 +1,13 @@
 [package]
 name = "opus"
-version = "0.1.0-alpha"
+version = "0.2.0-alpha"
+author = "xnacly"
+about = "manage tasks with ease"
```

Comment:
If you intend to display these on the help page, you'll still need to add them like this: https://docs.rs/clap/latest/clap/macro.crate_authors.html and https://docs.rs/clap/latest/clap/macro.crate_description.html